### ISSUE TITLE
Take over ships

### ISSUE THREAD

**Issue by v4nz666** [v4nz666/7drl2017#61]
If a ship's crew and captain are killed, you should be able to take the ship.

**Comment by Break-the-silence** [comment]
well we would need a plundered ship sound to go with that

**Comment by v4nz666** [comment]
Yeah... raucous hooting and hollering from the crew, I'd guess...
